lesson13/lesson13_script1.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# have two strings inside: Hello string and current date-time
+touch /home/dmitryyankov/lesson13/two.txt
```

Comment:
its bad practice to hardcode personal homepath, script wont work on another system

its better to create file_path variable

Reply:
fixed